sys/net/gnrc/network_layer/sixlowpan/nd/Kconfig

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_SIXLOWPAN_ND
+    bool "Configure GNRC 6LoWPAN Neighbor Discovery module"
+    depends on MODULE_GNRC_SIXLOWPAN_ND
+    help
+        Configure GNRC 6LoWPAN Neighbor Discovery module using Kconfig.
+
+if KCONFIG_MODULE_GNRC_SIXLOWPAN_ND
+
+config GNRC_SIXLOWPAN_ND_AR_LTIME
```

Comment:
Should we add a dependency on `MODULE_GNRC_IPV6_NIB`?

Reply:
More on `MODULE_GNRC_SIXLOWPAN_ND` I think.

Reply:
This already depends on `MODULE_GNRC_SIXLOWPAN_ND` because they are inside `if KCONFIG_MODULE_GNRC_SIXLOWPAN_ND`. My question was because the help string says:

>Only applicable with @ ref net_gnrc_ipv6_nib and @ ref
        net_gnrc_sixlowpan_nd modules.

Reply:
Ah, well those kind of comments in my doc usually mean "it has no effect if the module referred to is not compiled in", so I wouldn't really call it a dependency, but more of a condition.

Reply:
> it has no effect if the module referred to is not compiled in

Ok, so this means that without `gnrc_ipv6_nib` module this options is not applicable, so I think it should not show up.